spec/helpers/state_file/nj_tenant_eligibility_helper_spec.rb

```diff
@@ -83,36 +83,36 @@
     end
 
     describe "multiple checkbox interactions" do
-      context "when a hard no overrides a soft no" do
+      context "when ineligible overrides worksheet required" do
```

Comment:
[pebble] - i know you are working off the previous contexts, but i think this was a misuse of the text in `context` vs in `it`.

the fact that `ineligible` overrides `worksheet required` is part of the expectation for the `it`. I think that's already baked into `returns ineligible` but if desired could be specified like `"ineligible" overrides "worksheet required" and returns "ineligible"`, while the `context` is more that `taxpayer choose options that indicate both "ineligible" and "worksheet required"`.

Same would go for the other helper tests.

Reply:
Fixed!